### DISCUSSION TITLE
The sponsors reviewing takes too long (Vietnam)

### DISCUSSION THREAD

**Discussion by lucwj** [community/community#70612] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi Team,
I applied sponsors program for a while but did not receive any update.

My GitHub account: `lucduong`

Thanks,
Luc

**Comment by DHayes10** [comment, marked as answer]
Hi @lucduong, 

Thank you for reaching out. We typically review Sponsors accounts in the order they are received. We apologize about the delay as we have seen an influx of accounts due to our most recent international expansion. I have forwarded your request to our team for approval. You should see this reflected in your account within the next hour. 

Best, 
GitHub Sponsors

**Reply by lucwj** [reply]
Thank you so much for the quick turn over; I can see my account has been approved. <3
